packages/@storefront/navigation/src/value-refinement-controls/index.ts

```diff
@@ -36,7 +39,7 @@ class ValueRefinementControls extends RefinementControls<RefinementControls.Prop
       ...navigation,
       refinements: navigation.refinements.map((refinement) => ({
         ...refinement,
-        onClick: () => this.actions[refinement.selected ? this.deselectRefinement : this.selectRefinement](
+        onClick: () => this.actions[refinement.selected ? this.actionNames.deselect : this.actionNames.select](
```

Comment:
Calculate the respective action section on the fly, ie `this.actionNames[this.props.storeSection].deselect`, `this.actionNames[this.props.storeSection].select`